extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.vertx.http.deployment.devmode.console;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "dev-ui")
+public class DevConsoleConfig {
```

Comment:
DevConsole?

Reply:
Yea, so initially I started rename all DevConsole* to DevUI (see my prev [close PR](https://github.com/quarkusio/quarkus/pull/15536)) , but that seems to be a bigger effort and I think this is more important to get the thread safe issue solved. Renaming all things that is wrongly name Dev Console will also affect the extensions that build full pages for the DevUI. So I think we should do that in a later step. In that spirit I named this wrong then as well, to be consistent. I can open a ticket to fix the naming in a separate effort . w.d.y.t ?

Reply:
This is a new class though, I think it should match the name it exposes.

Reply:
Ok, renamed to DevUIConfig